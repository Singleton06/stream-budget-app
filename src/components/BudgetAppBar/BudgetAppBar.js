import AppBar from 'material-ui/AppBar';
import Toolbar from 'material-ui/Toolbar';
import Typography from 'material-ui/Typography';
import IconButton from 'material-ui/IconButton';
import { withStyles } from 'material-ui/styles';
import Icon from 'material-ui/Icon';
import SvgIcon from 'material-ui/SvgIcon';

import React from 'react';
import PropTypes from 'prop-types';

const styles = theme => ({
  appbar: {
    flexGrow: 1
  },
  dIcon: {},
  title: {
    flex: 1
  }
});

const BudgetAppBar = props => {
  const { classes } = props;
  return (
    <AppBar className={classes.flexGrow} position="static">
      <Toolbar>
        <IconButton className={classes.addIcon}>
          <Icon>add_circle_outline</Icon>
        </IconButton>
        <Typography className={classes.title} variant="title" color="inherit">
          {props.title}
        </Typography>
      </Toolbar>
    </AppBar>
  );
};

BudgetAppBar.propTypes = {
  title: PropTypes.string.isRequired
};

export default withStyles(styles)(BudgetAppBar);
