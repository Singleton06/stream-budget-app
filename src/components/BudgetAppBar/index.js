import BudgetAppBar from "./BudgetAppBar";

export default BudgetAppBar;
