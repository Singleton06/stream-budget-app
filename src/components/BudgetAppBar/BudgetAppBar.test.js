import React from 'react';
import renderer from 'react-test-renderer';

import BudgetAppBar from './BudgetAppBar';

describe('BudgetAppBar', () => {
  test('default props', () => {
    const tree = renderer.create(<BudgetAppBar />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  test('title prop specified', () => {
    const tree = renderer.create(<BudgetAppBar title="testing" />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});