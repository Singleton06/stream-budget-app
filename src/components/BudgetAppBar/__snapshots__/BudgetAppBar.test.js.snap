// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BudgetAppBar default props 1`] = `
<header
  className="MuiPaper-root-12 MuiPaper-elevation4-18 MuiAppBar-root-4 MuiAppBar-positionStatic-8 MuiAppBar-colorPrimary-10 BudgetAppBar-flexGrow-1"
>
  <div
    className="MuiToolbar-root-39 MuiToolbar-gutters-40"
  >
    <h2
      className="MuiTypography-root-41 MuiTypography-title-47 MuiTypography-colorInherit-60 BudgetAppBar-title-3"
    >
      
    </h2>
  </div>
</header>
`;

exports[`BudgetAppBar title prop specified 1`] = `
<header
  className="MuiPaper-root-12 MuiPaper-elevation4-18 MuiAppBar-root-4 MuiAppBar-positionStatic-8 MuiAppBar-colorPrimary-10 BudgetAppBar-flexGrow-1"
>
  <div
    className="MuiToolbar-root-39 MuiToolbar-gutters-40"
  >
    <h2
      className="MuiTypography-root-41 MuiTypography-title-47 MuiTypography-colorInherit-60 BudgetAppBar-title-3"
    >
      testing
    </h2>
  </div>
</header>
`;
